src/supervisedoutputconnector.h

```diff
@@ -1053,11 +1053,29 @@ namespace dd
             }
           if (beucll)
             {
-              double meucll = eucll(ad_res, -1);
+              std::tuple<double, std::vector<double>> tmeucll
+                  = eucll(ad_res, -1);
+              double meucll = std::get<0>(tmeucll);
```

Comment:
FYI possible syntax
```cpp
double meucll;
std::vector<double> all_meucll;
std::tie(meucll, all_meucll) = eucll(ad_res, -1);
```
C++ tuple unwrapping :) https://en.cppreference.com/w/cpp/utility/tuple/tie

Reply:
ok, it is much nicer :)

Reply:
![image](https://user-images.githubusercontent.com/3530657/99981160-56910680-2da9-11eb-8a66-87454c94691b.png)